src/fielddef.ts

```diff
@@ -177,13 +179,16 @@ export function title(fieldDef: FieldDef) {
   if (fieldDef.title != null) {
     return fieldDef.title;
   }
+  const unit = (fieldDef.unit && (fieldDef.unitPosition == 'title' || !fieldDef.unitPosition))
+                ? ' in ' + fieldDef.unit : '';
```

Comment:
Minor but I'd add some parens to make the flow clearer `...? (' in ' + fieldDef.unit) : '';...`